test/integration/dialects/sqlite/connection-manager.test.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
 const chai = require('chai');
-const fs = require('fs');
+const jetpack = require('fs-jetpack');
```

Comment:
I see that you made a few refactors on this file, that is great, I have a suggestion to make them even better though

```suggestion
const jetpack = require('fs-jetpack').cwd(__dirname);
```

This will let you make the `jetpack.remove` and `jetpack.exists` calls in this file even more super awesome...

Reply:
Awesome idea